.github/workflows/integration-tests.yaml

```diff
@@ -250,4 +250,13 @@ jobs:
             echo "Unknown mode: ${{ matrix.mode }}"
             exit 1
           fi
-          uvx tox -e integration -- -k "${{ matrix.suite }}"
\ No newline at end of file
+          uvx tox -e integration -- -k "${{ matrix.suite }}"
+
+  terraform-deploy-tests:
+    name: Run terraform deployment tests
+    uses: canonical/observability/.github/workflows/run-tox-environment.yaml@main
+    with:
+      tox-env-name: terraform-deploy
+      prepare-host-env: true
+      extra-snaps: terraform
```

Comment:
we had opentofu in the previous one no?

Reply:
Nope, that's different env. We use opentofu for formatting.